server/src/main/java/org/elasticsearch/transport/RemoteClusterAware.java

```diff
@@ -242,6 +243,10 @@ static DiscoveryNode buildSeedNode(String clusterName, String address, boolean p
                         perClusterIndices.computeIfAbsent(clusterName, k -> new ArrayList<>()).add(indexName);
                     }
                 } else {
+                    //Indices can be created with ":" in their names only up to 5.6, and still be there up to 6.last, but aliases may be
```

Comment:
I think both alias and index names can have `:` in the name until 6.last? https://github.com/elastic/elasticsearch/blob/6.x/server/src/main/java/org/elasticsearch/cluster/metadata/MetaDataCreateIndexService.java#L170

This validation method is both used for alias and index names. From 6.0 we log a deprecation warning, but not fail it. From 7.0 we fail such index or alias nems

Reply:
you are right, I thought that we introduced index name validation for that earlier compared to aliases. I will update.

Reply:
I pushed an update, thanks for double checking this Martijn.